mesonbuild/modules/pkgconfig.py

```diff
@@ -167,12 +174,12 @@ def generate_pkgconfig_file(self, state, deps, subdirs, name, description,
                 ofile.write('URL: %s\n' % url)
             ofile.write('Version: %s\n' % version)
             if len(deps.pub_reqs) > 0:
-                ofile.write('Requires: {}\n'.format(' '.join(deps.pub_reqs)))
+                ofile.write('Requires: {}\n'.format(' '.join(sorted(deps.pub_reqs))))
```

Comment:
What is the point of doing `_fn()` if you still call `sorted()` here? I think the idea to avoid `set()`/`sorted()` was to keep original order and remove duplicates.

Reply:
It was an error to keep this `sorted`.